README.md

```diff
@@ -1,9 +1,20 @@
-[![Build Status](https://travis-ci.org/oasis-roles/wipefs.svg?branch=master)](https://travis-ci.org/oasis-roles/wipefs)
+[![Build Status](https://travis-ci.org/oasis-roles/disk_wipe.svg?branch=master)](https://travis-ci.org/oasis-roles/disk_wipe)
 
-wipefs
+disk_wipe
 ===========
 
-Basic description for wipefs
+This role is not idempotent and is intended to be used when you are preparing nodes
+for deployment to ensure there is no stale data from previous deployments.
+It will perform the following 4 tasks:
+1. Ensure parted is installed.
+2. Erase the the disk signatures if detected from specified disks.
+3. Re-read the partition tables for specified disks.
+4. Completely erase all traces of existing data on specified disks.
```

Comment:
This would only be true if we were to `dd` or `shred` the entire disk from beginning to end.

Reply:
updated